internal/gps/constraints.go

```diff
@@ -52,7 +52,7 @@ type Constraint interface {
 // If the input string cannot be made into a valid semver Constraint, an error
 // is returned.
 func NewSemverConstraint(body string) (Constraint, error) {
```

Comment:
If I go that route, I can undo the changes to gps testdata, and have gps test without implied carets. So that's a win allaround! ❤️